### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by OnaMosimege**
Add custom error pages, with styling

### CODE REVIEW

File: app/templates/400.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
```
Comment (friedelwolff) [inline app/templates/400.html:1]: These error pages are largely the same. I suggest we put the common parts in a template and reuse them. That way these pages are tiny and a change to the design doesn't need to be replicated in all pages.

File: app/templates/400.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{% load static %}
```
Comment (friedelwolff) [inline app/templates/400.html:2]: If I check correctly, you don't need static here.

**Review by friedelwolff — CHANGES_REQUESTED** [review]
Left and right margins of the error cards are not equal.

Let's add a 500 page as well.

File: app/templates/400.html (hunk 2 above)
Comment (OnaMosimege) [reply app/templates/400.html:2]: Okay thanks, will remove

File: app/templates/400.html (hunk 1 above)
Comment (OnaMosimege) [reply app/templates/400.html:1]: Okay, I'll use template inheritance here

**Review by friedelwolff — APPROVED** [review]

**Pull request merged**